### ISSUE TITLE
Verify Modification.filename on Windows

### ISSUE THREAD

**Issue by avancinirodrigo** [ishepard/pydriller#13]
It seems that filename is returning a relative path from repository instead just the filename (with extension) on Windows. 

On Linux works correctly.

That happens using Python 3.5.1

To reproduce the problem:
 ```python
		for commit in RepositoryMining(repo_path, only_in_branches=['master'], 
					only_no_merge=True, from_tag=from_tag, 
					to_tag=to_tag).traverse_commits():
			for m in commit.modifications:
				print(m.filename)
```

**Comment by ishepard** [comment]
thank you! 
I am now using Appvoyer as CI to build PyDriller on Windows, and indeed it fails to get the filename. I am gonna have a look at it this weekend

**Comment by ishepard** [comment]
@avancinirodrigo I released PyDriller 1.4.3 with the fix. Now the filename should be correctly displayed on Windows machines.
